raft/src/main/java/org/apache/kafka/raft/Endpoints.java

```diff
@@ -101,6 +102,21 @@ public BeginQuorumEpochRequestData.LeaderEndpointCollection toBeginQuorumEpochRe
         return leaderEndpoints;
     }
 
+    public AddRaftVoterRequestData.ListenerCollection toAddVoterRequest() {
```

Comment:
It would be nice to have a unit test for these to / from methods.

Reply:
I agree. There are many Endpoints methods that are not unittested. Do you mind if I tackle this in a future PR? https://issues.apache.org/jira/browse/KAFKA-17169

Reply:
ok. thanks for filing the PR. we can ask if someone from the community is interested too.